packages/react-conformance/src/defaultTests.tsx

```diff
@@ -66,6 +66,33 @@ export const defaultTests: TestObject = {
     });
   },
 
+  /** Component file handles classname prop */
+  'component-contains-classname': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    it(`has a className prop`, () => {
+      const {
+        Component,
+        wrapperComponent,
+        helperComponents = [],
+        requiredProps,
+        customMount = mount,
+        passesUnhandledPropsTo,
+      } = testInfo;
+
+      const mergedProps: Partial<{}> = {
+        ...requiredProps,
+        className: 'testComponentClassName',
+      };
+
+      const el = passesUnhandledPropsTo
+        ? customMount(<Component {...mergedProps} />).find(passesUnhandledPropsTo)
+        : customMount(<Component {...mergedProps} />);
+
+      const component = getComponent(el, helperComponents, wrapperComponent);
+
+      expect(component.find('[testComponentClassName]'));
```

Comment:
This asserts nothing...

Reply:
In v0 we are testing at least one scenario: that passed `className` will not override a default one

```jsx
function Button(props) {
  return <button className='ui-button' {...props} />
}

<Button /> // has ui-button
<Button className="foo" /> // has only foo, that's not correct
```

Reply:
@layershifter Thanks for the suggestion! I just updated the pr to handle default classNames on components.